ExampleCode/C++/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#*****************************************************************************
+#  iSENSE C++ Class Install Script
+#*****************************************************************************
+
+printf "iSENSE C++ Class Install Script\n"
+printf "This script pulls all required dependencies of the iSENSE C++ Class\n"
+printf "Included are:\n"
+printf  "- Pulls down iSENSE class / includes\n"
+printf  "- Pulls down iSENSE examples / example apps\n"
+printf  "- Pulls down picojson - 3rd party JSON library that the class depends on\n"
+printf  "- Installs curl, libcurl\n"
+printf "Please only run this script once. When this script is finished, look for\n"
+printf "a directory named \"isense\" in your home directory (~/isense)\n"
```

Comment:
If there a reason why the user shouldn't install twice?  Will anything break?

If so, to avoid the user accidentally installing twice, in your `if` statement where you create the `~isense` directory, could you just have the script quit if it exists?  Maybe inform the user that you're doing this too so if they really meant to install it again, they know to delete `~isense`.